pkg/doc/jsonld/validate.go

```diff
@@ -46,6 +48,15 @@ func WithStrictValidation(checkStructure bool) ValidateOpts {
 	}
 }
 
+// WithStrictContextURIPosition sets strict validation of URI position within context property.
+// The index of uri in underlying slice represents the position of given uri in @context array.
+// Can be used for verifiable credential base context validation.
+func WithStrictContextURIPosition(uri string) ValidateOpts {
+	return func(opts *validateOpts) {
+		opts.contextURIPositions = append(opts.contextURIPositions, uri)
+	}
+}
```

Comment:
why do we need this flag?
By default JWT VCs should check presence of credential contexts as mentioned in #3351 .

And I don't see usage of existing `WithStrictValidation` option to structurally validate VCs.

Reply:
>why do we need this flag?

This flag is used to strictly define the position of base context withing contexts list.
I added it [here](https://github.com/hyperledger/aries-framework-go/pull/3357/files#diff-953974a5ec9fe3293be8ffd004be86b23666847d300650428cb21673e78fa140R889) to satisfy the documentation requirement:
```
The value of the @context MUST be an ordered set where the first item is a URI with the value 

https://www.w3.org/2018/credentials/v1.
```

>By default JWT VCs should check presence of credential contexts as mentioned in https://github.com/hyperledger/aries-framework-go/issues/3351 .

The default validation model is [combinedValidation](https://github.com/hyperledger/aries-framework-go/blob/main/pkg/doc/verifiable/credential.go#L830), that implies validation the VC against json schema and json ld validation (and they includes checking context property presence). Adding the [explicit requirement on base context](https://github.com/hyperledger/aries-framework-go/pull/3357/files#diff-953974a5ec9fe3293be8ffd004be86b23666847d300650428cb21673e78fa140R889) brings the requirement "make VC data model context as required and additional contexts can be optional". Without this addition strict validation option disabled will not return an error in case VC without base context URI.

>And I don't see usage of existing WithStrictValidation option to structurally validate VCs.

Strict validation option [is enabled by default](https://github.com/hyperledger/aries-framework-go/blob/main/pkg/doc/jsonld/validate.go#L51). Do you mean that is should be added explicitly over [here](https://github.com/hyperledger/aries-framework-go/pull/3357/files#diff-953974a5ec9fe3293be8ffd004be86b23666847d300650428cb21673e78fa140R889)?